Description: Upstream changes introduced in version 8.2.21-1~pgapt+1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 postgresql-8.2 (8.2.21-1~pgapt+1) sid-pgapt; urgency=low
 .
   * Backport packaging to 8.2.
 .
 The person named in the Author field signed this changelog entry.
Author: Christoph Berg <myon@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- postgresql-8.2-8.2.21.orig/src/interfaces/libpq/libpq.rc
+++ postgresql-8.2-8.2.21/src/interfaces/libpq/libpq.rc
@@ -1,8 +1,8 @@
 #include <winver.h>
 
 VS_VERSION_INFO VERSIONINFO
- FILEVERSION 8,2,21,11105
- PRODUCTVERSION 8,2,21,11105
+ FILEVERSION 8,2,21,11265
+ PRODUCTVERSION 8,2,21,11265
  FILEFLAGSMASK 0x3fL
  FILEFLAGS 0
  FILEOS VOS__WINDOWS32
--- /dev/null
+++ postgresql-8.2-8.2.21/src/backend/gettext-files
@@ -0,0 +1,484 @@
+./access/nbtree/nbtree.c
+./access/nbtree/nbtinsert.c
+./access/nbtree/nbtxlog.c
+./access/nbtree/nbtsort.c
+./access/nbtree/nbtcompare.c
+./access/nbtree/nbtsearch.c
+./access/nbtree/nbtpage.c
+./access/nbtree/nbtutils.c
+./access/gin/ginutil.c
+./access/gin/ginvacuum.c
+./access/gin/gininsert.c
+./access/gin/gindatapage.c
+./access/gin/ginbulk.c
+./access/gin/ginbtree.c
+./access/gin/ginscan.c
+./access/gin/ginxlog.c
+./access/gin/ginget.c
+./access/gin/ginentrypage.c
+./access/gin/ginarrayproc.c
+./access/common/printtup.c
+./access/common/indextuple.c
+./access/common/heaptuple.c
+./access/common/reloptions.c
+./access/common/tupdesc.c
+./access/common/scankey.c
+./access/hash/hashscan.c
+./access/hash/hashsearch.c
+./access/hash/hash.c
+./access/hash/hashinsert.c
+./access/hash/hashpage.c
+./access/hash/hashfunc.c
+./access/hash/hashutil.c
+./access/hash/hashovfl.c
+./access/gist/gistproc.c
+./access/gist/gistget.c
+./access/gist/gistvacuum.c
+./access/gist/gist.c
+./access/gist/gistxlog.c
+./access/gist/gistsplit.c
+./access/gist/gistutil.c
+./access/gist/gistscan.c
+./access/heap/hio.c
+./access/heap/heapam.c
+./access/heap/tuptoaster.c
+./access/transam/xlogutils.c
+./access/transam/varsup.c
+./access/transam/multixact.c
+./access/transam/subtrans.c
+./access/transam/xlog.c
+./access/transam/rmgr.c
+./access/transam/xact.c
+./access/transam/slru.c
+./access/transam/clog.c
+./access/transam/transam.c
+./access/transam/twophase.c
+./access/transam/twophase_rmgr.c
+./access/index/indexam.c
+./access/index/genam.c
+./regex/regc_cvec.c
+./regex/rege_dfa.c
+./regex/regc_lex.c
+./regex/regc_nfa.c
+./regex/regfree.c
+./regex/regerror.c
+./regex/regcomp.c
+./regex/regexec.c
+./regex/regc_locale.c
+./regex/regc_color.c
+./storage/smgr/smgrtype.c
+./storage/smgr/smgr.c
+./storage/smgr/md.c
+./storage/freespace/freespace.c
+./storage/page/bufpage.c
+./storage/page/itemptr.c
+./storage/ipc/sinvaladt.c
+./storage/ipc/ipc.c
+./storage/ipc/sinval.c
+./storage/ipc/procarray.c
+./storage/ipc/ipci.c
+./storage/ipc/shmem.c
+./storage/ipc/shmqueue.c
+./storage/ipc/pmsignal.c
+./storage/lmgr/spin.c
+./storage/lmgr/s_lock.c
+./storage/lmgr/lwlock.c
+./storage/lmgr/deadlock.c
+./storage/lmgr/lock.c
+./storage/lmgr/lmgr.c
+./storage/lmgr/proc.c
+./storage/buffer/buf_table.c
+./storage/buffer/freelist.c
+./storage/buffer/localbuf.c
+./storage/buffer/buf_init.c
+./storage/buffer/bufmgr.c
+./storage/file/fd.c
+./storage/file/buffile.c
+./storage/large_object/inv_api.c
+./parser/parse_func.c
+./parser/parse_clause.c
+./parser/gram.c
+./parser/parse_target.c
+./parser/parse_expr.c
+./parser/parse_relation.c
+./parser/parse_type.c
+./parser/parse_node.c
+./parser/parse_agg.c
+./parser/parse_oper.c
+./parser/analyze.c
+./parser/keywords.c
+./parser/parse_coerce.c
+./parser/parser.c
+./parser/scansup.c
+./parser/scan.c
+./lib/dllist.c
+./lib/stringinfo.c
+./tcop/fastpath.c
+./tcop/postgres.c
+./tcop/utility.c
+./tcop/pquery.c
+./tcop/dest.c
+./main/main.c
+./executor/nodeSubplan.c
+./executor/nodeBitmapAnd.c
+./executor/nodeMergejoin.c
+./executor/nodeNestloop.c
+./executor/nodeMaterial.c
+./executor/nodeTidscan.c
+./executor/execQual.c
+./executor/nodeSort.c
+./executor/nodeBitmapIndexscan.c
+./executor/tstoreReceiver.c
+./executor/nodeAgg.c
+./executor/execTuples.c
+./executor/nodeIndexscan.c
+./executor/nodeSetOp.c
+./executor/nodeFunctionscan.c
+./executor/spi.c
+./executor/nodeSubqueryscan.c
+./executor/nodeLimit.c
+./executor/nodeBitmapOr.c
+./executor/execAmi.c
+./executor/nodeValuesscan.c
+./executor/nodeBitmapHeapscan.c
+./executor/execScan.c
+./executor/nodeResult.c
+./executor/nodeHash.c
+./executor/nodeGroup.c
+./executor/functions.c
+./executor/nodeSeqscan.c
+./executor/nodeAppend.c
+./executor/nodeUnique.c
+./executor/execProcnode.c
+./executor/execJunk.c
+./executor/execUtils.c
+./executor/execMain.c
+./executor/execGrouping.c
+./executor/instrument.c
+./executor/nodeHashjoin.c
+./postmaster/fork_process.c
+./postmaster/pgarch.c
+./postmaster/postmaster.c
+./postmaster/pgstat.c
+./postmaster/syslogger.c
+./postmaster/autovacuum.c
+./postmaster/bgwriter.c
+./bootstrap/bootscanner.c
+./bootstrap/bootstrap.c
+./bootstrap/bootparse.c
+./port/ipc_test.c
+./port/win32_sema.c
+./port/sysv_shmem.c
+./port/posix_sema.c
+./port/win32/timer.c
+./port/win32/signal.c
+./port/win32/security.c
+./port/win32/socket.c
+./port/win32/shmem.c
+./port/pg_shmem.c
+./port/nextstep/port.c
+./port/darwin/system.c
+./port/dynloader.c
+./port/sysv_sema.c
+./port/pg_sema.c
+./port/dynloader/sco.c
+./port/dynloader/sunos4.c
+./port/dynloader/nextstep.c
+./port/dynloader/ultrix4.c
+./port/dynloader/svr4.c
+./port/dynloader/freebsd.c
+./port/dynloader/win32.c
+./port/dynloader/cygwin.c
+./port/dynloader/openbsd.c
+./port/dynloader/aix.c
+./port/dynloader/irix.c
+./port/dynloader/netbsd.c
+./port/dynloader/osf.c
+./port/dynloader/linux.c
+./port/dynloader/univel.c
+./port/dynloader/dgux.c
+./port/dynloader/unixware.c
+./port/dynloader/hpux.c
+./port/dynloader/bsdi.c
+./port/dynloader/solaris.c
+./port/dynloader/darwin.c
+./rewrite/rewriteManip.c
+./rewrite/rewriteDefine.c
+./rewrite/rewriteHandler.c
+./rewrite/rewriteRemove.c
+./rewrite/rewriteSupport.c
+./optimizer/path/costsize.c
+./optimizer/path/indxpath.c
+./optimizer/path/orindxpath.c
+./optimizer/path/clausesel.c
+./optimizer/path/pathkeys.c
+./optimizer/path/allpaths.c
+./optimizer/path/tidpath.c
+./optimizer/path/joinpath.c
+./optimizer/path/joinrels.c
+./optimizer/geqo/geqo_ox2.c
+./optimizer/geqo/geqo_erx.c
+./optimizer/geqo/geqo_misc.c
+./optimizer/geqo/geqo_px.c
+./optimizer/geqo/geqo_pool.c
+./optimizer/geqo/geqo_cx.c
+./optimizer/geqo/geqo_copy.c
+./optimizer/geqo/geqo_eval.c
+./optimizer/geqo/geqo_selection.c
+./optimizer/geqo/geqo_ox1.c
+./optimizer/geqo/geqo_mutation.c
+./optimizer/geqo/geqo_main.c
+./optimizer/geqo/geqo_recombination.c
+./optimizer/geqo/geqo_pmx.c
+./optimizer/util/predtest.c
+./optimizer/util/plancat.c
+./optimizer/util/restrictinfo.c
+./optimizer/util/tlist.c
+./optimizer/util/var.c
+./optimizer/util/clauses.c
+./optimizer/util/pathnode.c
+./optimizer/util/joininfo.c
+./optimizer/util/relnode.c
+./optimizer/prep/prepjointree.c
+./optimizer/prep/preptlist.c
+./optimizer/prep/prepqual.c
+./optimizer/prep/prepunion.c
+./optimizer/plan/setrefs.c
+./optimizer/plan/planagg.c
+./optimizer/plan/planmain.c
+./optimizer/plan/planner.c
+./optimizer/plan/createplan.c
+./optimizer/plan/initsplan.c
+./optimizer/plan/subselect.c
+./commands/async.c
+./commands/dbcommands.c
+./commands/explain.c
+./commands/define.c
+./commands/typecmds.c
+./commands/aggregatecmds.c
+./commands/cluster.c
+./commands/prepare.c
+./commands/tablecmds.c
+./commands/vacuum.c
+./commands/sequence.c
+./commands/schemacmds.c
+./commands/tablespace.c
+./commands/proclang.c
+./commands/copy.c
+./commands/conversioncmds.c
+./commands/portalcmds.c
+./commands/view.c
+./commands/analyze.c
+./commands/variable.c
+./commands/indexcmds.c
+./commands/user.c
+./commands/vacuumlazy.c
+./commands/trigger.c
+./commands/opclasscmds.c
+./commands/functioncmds.c
+./commands/comment.c
+./commands/alter.c
+./commands/operatorcmds.c
+./commands/lockcmds.c
+./catalog/namespace.c
+./catalog/pg_depend.c
+./catalog/heap.c
+./catalog/catalog.c
+./catalog/pg_constraint.c
+./catalog/toasting.c
+./catalog/pg_operator.c
+./catalog/pg_conversion.c
+./catalog/index.c
+./catalog/pg_aggregate.c
+./catalog/pg_largeobject.c
+./catalog/pg_proc.c
+./catalog/dependency.c
+./catalog/pg_namespace.c
+./catalog/pg_type.c
+./catalog/indexing.c
+./catalog/pg_shdepend.c
+./catalog/aclchk.c
+./utils/misc/superuser.c
+./utils/misc/ps_status.c
+./utils/misc/guc-file.c
+./utils/misc/help_config.c
+./utils/misc/pg_rusage.c
+./utils/misc/tzparser.c
+./utils/misc/guc.c
+./utils/mb/win866.c
+./utils/mb/conversion_procs/utf8_and_sjis/utf8_and_sjis.c
+./utils/mb/conversion_procs/utf8_and_uhc/utf8_and_uhc.c
+./utils/mb/conversion_procs/utf8_and_cyrillic/utf8_and_cyrillic.c
+./utils/mb/conversion_procs/utf8_and_gb18030/utf8_and_gb18030.c
+./utils/mb/conversion_procs/ascii_and_mic/ascii_and_mic.c
+./utils/mb/conversion_procs/utf8_and_euc_jp/utf8_and_euc_jp.c
+./utils/mb/conversion_procs/utf8_and_big5/utf8_and_big5.c
+./utils/mb/conversion_procs/euc_tw_and_big5/big5.c
+./utils/mb/conversion_procs/euc_tw_and_big5/euc_tw_and_big5.c
+./utils/mb/conversion_procs/utf8_and_euc_kr/utf8_and_euc_kr.c
+./utils/mb/conversion_procs/utf8_and_ascii/utf8_and_ascii.c
+./utils/mb/conversion_procs/utf8_and_euc_tw/utf8_and_euc_tw.c
+./utils/mb/conversion_procs/utf8_and_euc_cn/utf8_and_euc_cn.c
+./utils/mb/conversion_procs/euc_kr_and_mic/euc_kr_and_mic.c
+./utils/mb/conversion_procs/euc_jp_and_sjis/euc_jp_and_sjis.c
+./utils/mb/conversion_procs/utf8_and_iso8859_1/utf8_and_iso8859_1.c
+./utils/mb/conversion_procs/utf8_and_gbk/utf8_and_gbk.c
+./utils/mb/conversion_procs/utf8_and_johab/utf8_and_johab.c
+./utils/mb/conversion_procs/latin2_and_win1250/latin2_and_win1250.c
+./utils/mb/conversion_procs/cyrillic_and_mic/cyrillic_and_mic.c
+./utils/mb/conversion_procs/utf8_and_win/utf8_and_win.c
+./utils/mb/conversion_procs/euc_cn_and_mic/euc_cn_and_mic.c
+./utils/mb/conversion_procs/latin_and_mic/latin_and_mic.c
+./utils/mb/conversion_procs/utf8_and_iso8859/utf8_and_iso8859.c
+./utils/mb/wchar.c
+./utils/mb/win1251.c
+./utils/mb/mbutils.c
+./utils/mb/wstrncmp.c
+./utils/mb/conv.c
+./utils/mb/wstrcmp.c
+./utils/mb/encnames.c
+./utils/mb/iso.c
+./utils/init/postinit.c
+./utils/init/flatfiles.c
+./utils/init/globals.c
+./utils/init/miscinit.c
+./utils/cache/typcache.c
+./utils/cache/lsyscache.c
+./utils/cache/relcache.c
+./utils/cache/inval.c
+./utils/cache/catcache.c
+./utils/cache/syscache.c
+./utils/hash/hashfn.c
+./utils/hash/pg_crc.c
+./utils/hash/dynahash.c
+./utils/adt/inet_net_pton.c
+./utils/adt/inet_net_ntop.c
+./utils/adt/like_match.c
+./utils/adt/like.c
+./utils/adt/bool.c
+./utils/adt/formatting.c
+./utils/adt/ruleutils.c
+./utils/adt/regproc.c
+./utils/adt/geo_ops.c
+./utils/adt/char.c
+./utils/adt/domains.c
+./utils/adt/format_type.c
+./utils/adt/int.c
+./utils/adt/pg_locale.c
+./utils/adt/date.c
+./utils/adt/tid.c
+./utils/adt/float.c
+./utils/adt/varbit.c
+./utils/adt/varlena.c
+./utils/adt/quote.c
+./utils/adt/ri_triggers.c
+./utils/adt/numeric.c
+./utils/adt/network.c
+./utils/adt/numutils.c
+./utils/adt/encode.c
+./utils/adt/varchar.c
+./utils/adt/arrayfuncs.c
+./utils/adt/int8.c
+./utils/adt/datum.c
+./utils/adt/not_in.c
+./utils/adt/nabstime.c
+./utils/adt/pg_lzcompress.c
+./utils/adt/xid.c
+./utils/adt/selfuncs.c
+./utils/adt/array_userfuncs.c
+./utils/adt/pgstatfuncs.c
+./utils/adt/regexp.c
+./utils/adt/mac.c
+./utils/adt/genfile.c
+./utils/adt/oid.c
+./utils/adt/oracle_compat.c
+./utils/adt/acl.c
+./utils/adt/datetime.c
+./utils/adt/timestamp.c
+./utils/adt/misc.c
+./utils/adt/version.c
+./utils/adt/arrayutils.c
+./utils/adt/pseudotypes.c
+./utils/adt/rowtypes.c
+./utils/adt/geo_selfuncs.c
+./utils/adt/dbsize.c
+./utils/adt/ascii.c
+./utils/adt/cash.c
+./utils/adt/name.c
+./utils/adt/lockfuncs.c
+./utils/error/assert.c
+./utils/error/elog.c
+./utils/time/tqual.c
+./utils/sort/tuplesort.c
+./utils/sort/tuplestore.c
+./utils/sort/logtape.c
+./utils/fmgr/dfmgr.c
+./utils/fmgr/fmgr.c
+./utils/fmgr/funcapi.c
+./utils/resowner/resowner.c
+./utils/fmgrtab.c
+./utils/mmgr/mcxt.c
+./utils/mmgr/aset.c
+./utils/mmgr/portalmem.c
+./nodes/print.c
+./nodes/readfuncs.c
+./nodes/makefuncs.c
+./nodes/read.c
+./nodes/nodes.c
+./nodes/value.c
+./nodes/equalfuncs.c
+./nodes/nodeFuncs.c
+./nodes/list.c
+./nodes/outfuncs.c
+./nodes/tidbitmap.c
+./nodes/params.c
+./nodes/copyfuncs.c
+./nodes/bitmapset.c
+./libpq/pqformat.c
+./libpq/hba.c
+./libpq/crypt.c
+./libpq/auth.c
+./libpq/be-secure.c
+./libpq/pqsignal.c
+./libpq/ip.c
+./libpq/pqcomm.c
+./libpq/md5.c
+./libpq/be-fsstubs.c
+./../port/gettimeofday.c
+./../port/crypt.c
+./../port/kill.c
+./../port/getrusage.c
+./../port/qsort.c
+./../port/thread.c
+./../port/unsetenv.c
+./../port/exec.c
+./../port/qsort_arg.c
+./../port/copydir.c
+./../port/snprintf.c
+./../port/rint.c
+./../port/memcmp.c
+./../port/getaddrinfo.c
+./../port/getopt.c
+./../port/srandom.c
+./../port/getopt_long.c
+./../port/strdup.c
+./../port/noblock.c
+./../port/dirmod.c
+./../port/pgsleep.c
+./../port/gethostname.c
+./../port/path.c
+./../port/strtol.c
+./../port/fseeko.c
+./../port/rand.c
+./../port/random.c
+./../port/win32error.c
+./../port/pgstrcasecmp.c
+./../port/isinf.c
+./../port/strtoul.c
+./../port/sprompt.c
+./../port/strlcpy.c
+./../port/open.c
+./../port/dirent.c
+./../port/pipe.c
+./../port/inet_aton.c
+./../port/strerror.c
